seisbench/data/geofon.py

```diff
@@ -156,6 +244,11 @@ def _get_event_params(event):
                 event_params["source_focal_mechanism_n_plunge"] = n_axis.plunge
                 event_params["source_focal_mechanism_n_length"] = n_axis.length
             except AttributeError:
+                print(
```

Comment:
Please use `seisbench.logger` instead of print.

Reply:
That makes a lot of sense